flink-end-to-end-tests/test-scripts/common_s3_minio.sh

```diff
@@ -107,4 +107,10 @@ function s3_setup_with_provider {
   set_config_key "s3.path-style-access" "true"
 }
 
+###################################
+# Simulates fetching the result in part files into the OUTPUT_PATH. Minio volume is actually mapped, such that it will
+# already write at the correct location. This function ensures compability with common_s3.sh script.
+###################################
+function fetch_complete_result { :; }
```

Comment:
What happens if I source common_s3 after this script?

Reply:
Most importantly, the setup methods are overridden, so there is no way to interact with the container anymore. The docker container should still be cleaned up.
Like I pointed out in a previous comment, it's meant as a drop-in replacement. You can run the `test_streaming_file_sink.sh` on minio by simply changing from `common_s3.sh` the import to `common_s3_minio.sh`.